src/repositories/mutable_repository.cr

```diff
@@ -0,0 +1,34 @@
+module BrokenCrystal
+    class MutableRepository
+        @db: DBWrapper
+
+        def initialize(@table : String)
+        end
+
+        def insert(insert_columns : Array(String), values : Array)
+            @db.query_runner.query("INSERT INTO #{@table}(#{insert_columns.join(", ")}) VALUES(#{values.join(", ")})")
+        end
+
+        def updateAll(update_columns : Array(String), values : String)
```

Comment:
method names should be underscore style
should be: `update_all`
https://crystal-lang.org/reference/conventions/coding_style.html